src/feature-test/user_see_form.test.js

```diff
@@ -3,6 +3,7 @@ import '../setupTest.js'
 
 describe('I.I.A.S', () => {
   it('the user can upload recipes inputing ingredients', async () => {
+    jest.setTimeout(30000)
     const browser = await puppeteer.launch({
       slowMo: 40,
```

Comment:
You don't need this.
This slowMo does what it says - it slows things down. Generally used in conjunction with headless being `false`.
I don't believe this is necessary.

Reply:
Resolve, I took it out